test/unit/units/prepare_test.py

```diff
@@ -152,7 +185,7 @@ def test_main(
             )
         ]
         mock_Path.call_args_list == [
-            call(['/var/lib/cloudregister']),
+            call(['/var/cache/cloudregister']),
```

Comment:
did the test for `/var/lib/cloudregister` disappear ? 
If so, should we not make sure that is still OK ?